snapcraft.yaml

```diff
@@ -16,10 +16,11 @@ parts:
     source: https://github.com/juju/juju.git
     source-type: git
     build-packages: [gcc]
-    #If you want to grab a specific revision tag, include it here
```

Comment:
Similarly here, if I'm developing a feature branch, and I want to use my CWD, is there a way to trigger that?